### DISCUSSION TITLE
Latest stable release not building on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by odesenfans** [tianocore/edk2#2988] [Q&A] [answered]
Hello,

I am attempting to build OVMF on Ubuntu 20.04. I followed this flow, as mentioned [here](https://www.linux-kvm.org/downloads/lersek/ovmf-whitepaper-c770f8c.txt):

```
git clone https://github.com/tianocore/edk2.git
cd edk2
git checkout edk2-stable202205
git submodule update --init
OvmfPkg/build.sh -a X64
```

This results in the following (cryptic) errors: [edk2-stable202205-build.log](https://github.com/tianocore/edk2/files/8933260/edk2-stable202205-build.log).

Realizing these instructions might be outdated, I also tried the method recommended [on Github](https://github.com/tianocore/tianocore.github.io/wiki/How-to-build-OVMF), with the same result.

Switching to `edk2-stable202202` works.

My config:
* Ubuntu 20.04
* GCC 9.4.0
* iasl 20190509
* nasm 2.14.02

Any idea?

**Comment by osteffenrh** [comment, marked as answer]
HI Olivier,

please try with the latest version of nasm:
https://www.nasm.us/pub/nasm/releasebuilds/2.15.05/

- Oliver

**Reply by odesenfans** [reply]
Seems to work, thanks!
